Dockerfile.example

```diff
@@ -0,0 +1,29 @@
+# Build
+FROM golang:1.12-alpine3.11 AS build
```

Comment:
any reasont o not use golang:1 ?

Reply:
I'm not a go-lang programmer, just saw this - https://github.com/pquerna/otp/blob/master/go.mod#L1-L4 so I thought it's best to be compatible, should I set it to 1.15? 1.16?
![image](https://user-images.githubusercontent.com/15122452/103939231-2375bd00-5134-11eb-935a-5555f6ad5176.png)